sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -115,44 +146,126 @@ class AdminOperations {
+            completion!(announcement)
+            }) { (operation, error) -> Void in
+                failure("Could not edit announcement.")
+        }
     }
     
     static func approveCheckin(checkin: Checkin, completion: (() -> Void)?, failure: (String) -> Void) {
         let manager = BaseOperation.manager()
-        // TODO: make a network request
+        
+        let checkinURLString = StringConstants.kCheckinAdminVerifyURL(checkin.id)
+        
+        manager.POST(checkinURLString, parameters: nil, success: { (operation, data) -> Void in
+            completion?()
+            }) { (operation, error) -> Void in
+                failure("Could not approve checkin.")
+        }
     }
     
     static func removeCheckin(checkin: Checkin, completion: (() -> Void)?, failure: (String) -> Void) {
         let manager = BaseOperation.manager()
-        // TODO: make a network request
+        let checkinURLString = StringConstants.kCheckinAdminDetailURL(checkin.id)
+        
+        manager.DELETE(checkinURLString, parameters: nil, success: { (operation, data) -> Void in
+            completion?()
+            }) { (operation, error) -> Void in
+                failure("Could not remove checkin.")
+        }
+
     }
     
     static func verifyUser(user: User, completion: (() -> Void)?, failure: (String) -> Void) {
+        // TODO: fix, currently doesn't verify
         let manager = BaseOperation.manager()
-        // TODO: make a network request
+
+        let userURLString = StringConstants.kUserAdminVerifyURL(user.id)
+        manager.POST(userURLString, parameters: nil, success: { (operation, data) -> Void in
+            completion?()
+            }) { (operation, error) -> Void in
+                failure("Couldn't verify user.")
+        }
     }
     
     static func removeUser(user: User, completion: (() -> Void)?, failure: (String) -> Void) {
         let manager = BaseOperation.manager()
-        // TODO: make a network request
+        let userURLString = StringConstants.kUserDetailURL(user.id)
+        manager.DELETE(userURLString, parameters: nil, success: { (operation, data) -> Void in
+            completion?()
+            }) { (operation, error) -> Void in
+                failure("Could not remove user.")
+        }
     }
     
     static func createAnnouncement(announcement: Announcement, completion: (Announcement) -> Void, failure: (String) -> Void) {
         let manager = BaseOperation.manager()
-        let announcementDict = announcement.toDictionary()
-        // TODO: Create Announcement
+        let params = ["announcement":
+            [
+                AnnouncementConstants.kTitle: announcement.title!,
+                AnnouncementConstants.kText: announcement.text!,
+                AnnouncementConstants.kSchoolID: announcement.school!.id,
+                AnnouncementConstants.kUserID: LoginOperations.getUser()!.id,
+                AnnouncementConstants.kCategory: "school"
```

Comment:
id think just not passing a school id is the move...but idk about the category part